niworkflows/func/util.py

```diff
@@ -104,10 +114,12 @@ def init_bold_reference_wf(
 
     """
     workflow = Workflow(name=name)
-    workflow.__desc__ = """\
+    workflow.__desc__ = f"""\
 First, a reference volume and its skull-stripped version were generated
-using a custom methodology of *fMRIPrep*.
+{'from the shortest echo of the BOLD run' * multiecho} using a custom
```

Comment:
It seems poldracklab/fmriprep#1803 is not picking this multi-echo note. Will follow-up when I catch the issue.

Reply:
Fixed.